testnet/stacks-node/src/burnchains/bitcoin_regtest_controller.rs

```diff
@@ -162,6 +162,13 @@ pub fn make_bitcoin_indexer(config: &Config) -> BitcoinIndexer {
     burnchain_indexer
 }
 
+pub fn get_satoshis_per_byte(config: Config) -> u64 {
```

Comment:
Do you think this could be an instance method of `Config`, and take `&self`?

Reply:
There's one usage on the `LeaderBlockCommitFees` impl that prevented me from doing this